packages/zmarkdown/index.js

```diff
@@ -78,6 +98,12 @@ const zmdParser = (config) => {
         file.data.disableToc = false
       })
     })
+    .use(() => (tree) => {
+      Object.keys(wrappers).forEach(
```

Comment:
HAST nodes should be wrapped, not MDAST nodes. This modification should only be applied in rehype for two reasons: 

* there's no such thing as `div`s in Markdown
* we only care about wrapping things in HTML because we are only wrapping for CSS styling purpose

Reply:
yes you're right